From 910a383f0b1719709090afae162dbd0963469429 Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 22 Mar 2023 15:35:11 +0100
Subject: [PATCH 4/6] Export GZ_VERSION in rules file

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index 2ab7b14..1afb05c 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -8,6 +8,7 @@
 
 # Uncomment this to turn on verbose mode.
 export DH_VERBOSE=1
+export GZ_VERSION=garden
 # TODO: remove the LDFLAGS override.  It's here to avoid esoteric problems
 # of this sort:
 #  https://code.ros.org/trac/ros/ticket/2977
-- 
2.34.1

